src/components/EmployeesSearch/EmployeesSearch.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { MainField } from '../UI/Fields/MainField';
+import { Icon } from '../UI/Icons/Icons';
+import { MainButton } from '../UI/Buttons/MainButton';
+import { searchSchema } from '../../utils/ValidationSchema';
+
+export default function EmployeesSearch({
+  setIsSearchFiltration,
+  isSearchFiltration,
+}) {
+  const [isFocusedSearch, setIsFocusedSearch] = useState(false);
+  const [isInputIconPosition, setInputIconPosition] =
```

Comment:
```suggestion
  const [inputIconPosition, setInputIconPosition] =
```
с is начинаются только булевы переменные